wpilibj/src/main/java/edu/wpi/first/wpilibj/kinematics/DifferentialDriveWheelSpeeds.java

```diff
@@ -59,4 +59,10 @@ public void normalize(double attainableMaxSpeedMetersPerSecond) {
           * attainableMaxSpeedMetersPerSecond;
     }
   }
+
+  @Override
+  public String toString() {
+    return String.format("DifferentialDriveWheelSpeeds(Left: %.2f m/s, Right: %.2f m/s)",
```

Comment:
Your capitalization is inconsistent. Pick uppercase or lowercase. I'd prefer the latter personally.